Readability.js

```diff
@@ -1743,12 +1743,15 @@ Readability.prototype = {
         return false;
       }
 
-      var textContentLength = node.textContent.trim().length;
-      if (textContentLength < 140) {
-        return false;
+      // In some web pages, they use "<br>" to typeset articles.
+      if (node.tagName === "BR") {
+        score += 2;
```

Comment:
So there are 2 problems with this.

1. This will mean that any page that includes more than 10 `<br>` tags will be marked as 'readerable' (so they'll get a reader mode icon). That doesn't seem like a good idea.
1. Performance. This code runs when we load any webpage. It needs to be as light-weight as possible.